python-avd/pyavd/_cv/workflows/finalize_workspace_on_cv.py

```diff
@@ -65,9 +65,35 @@ async def finalize_workspace_on_cv(workspace: CVWorkspace, cv_client: CVClient)
             workspace_id=workspace.id,
             request_id=workspace_config.request_params.request_id,
         )
+        # Form a list of known inactive existing devices
+        if inactive_devices := [device for device in devices if not device._streaming]:
```

Comment:
The print-out of inactive devices is not very user-friendly. Let's do a string instead.

```suggestion
        if inactive_devices := [f"{device.hostname}({device.serial_number})" for device in devices if not device._streaming]:
```

Reply:
Fixed